<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Update_Resultats_Equipes</class>
 <widget class="QDialog" name="Update_Resultats_Equipes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des sportifs</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="radioInsert">
       <property name="text">
        <string>Insérer</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioUpdate">
       <property name="text">
        <string>Suppri&amp;mer</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_insert_num">
         <property name="text">
          <string>Numéro du nouveau sportif :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spin_insert_num">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimum">
          <number>1000</number>
         </property>
         <property name="maximum">
          <number>1500</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_insert_nom">
         <property name="text">
          <string>Nom :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="line_insert_nom"/>
       </item>
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_insert_prenom">
         <property name="text">
          <string>Prénom :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="line_insert_prenom"/>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_insert_pays">
         <property name="text">
          <string>Pays :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="line_insert_pays"/>
       </item>
       <item>
        <spacer name="horizontalSpacer_7">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_insert_categorie">
         <property name="text">
          <string>Catégorie :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="combo_insert_categorie">
         <item>
          <property name="text">
           <string>feminin</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>masculin</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_8">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_insert_date_nais">
         <property name="text">
          <string>Date de naissance (jj/mm/aaaa):</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="date_insert_naissance">
         <property name="locale">
          <locale language="French" country="France"/>
         </property>
         <property name="displayFormat">
          <string>dd/MM/yyyy</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_10">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_delete">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Sportif à supprimer :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_delete_recap">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableWidget" name="table_delete">
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <column>
        <property name="text">
         <string>numSp</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>nomSp</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>prenomSp</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>pays</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>categorieSp</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>dateNaisSp</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>ageSp</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_Error">
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Bouton_Validation">
       <property name="text">
        <string>Valider</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>radioInsert</sender>
   <signal>clicked()</signal>
   <receiver>Update_Resultats_Equipes</receiver>
   <slot>setGoodLayout()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>100</x>
     <y>14</y>
    </hint>
    <hint type="destinationlabel">
     <x>192</x>
     <y>18</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioUpdate</sender>
   <signal>clicked()</signal>
   <receiver>Update_Resultats_Equipes</receiver>
   <slot>setGoodLayout()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>477</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>610</x>
     <y>15</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>table_delete</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>Update_Resultats_Equipes</receiver>
   <slot>setLabelRecap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>147</x>
     <y>318</y>
    </hint>
    <hint type="destinationlabel">
     <x>-4</x>
     <y>316</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Bouton_Validation</sender>
   <signal>clicked()</signal>
   <receiver>Update_Resultats_Equipes</receiver>
   <slot>validate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>520</x>
     <y>426</y>
    </hint>
    <hint type="destinationlabel">
     <x>439</x>
     <y>404</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>refreshResult()</slot>
  <slot>refreshEquipesList()</slot>
  <slot>refresh2emeplaceList()</slot>
  <slot>refresh3emeplaceList()</slot>
  <slot>refreshEpreuvesList()</slot>
  <slot>setGoodLayout()</slot>
  <slot>setLabelRecap()</slot>
  <slot>validate()</slot>
 </slots>
</ui>
